src/infrastructure/chatwoot/sync.go

```
@@ -623,9 +899,12 @@ func (s *SyncService) syncMessageWithOptions(
 	}
 
 	if err != nil {
-		return fmt.Errorf("failed to create message: %w", err)
+		return false, fmt.Errorf("failed to create message: %w", err)
 	}
 
+	// From here on a message exists in Chatwoot whatever happens to the link
+	// store below; callers use posted to decide whether to reopen the thread.
+	posted = true
```

golangci-lint (2.13.2)

[error] ineffectual assignment to posted

(ineffassign)